schema/schema.json

```diff
@@ -0,0 +1,348 @@
+                    "type": "string",
+                    "enum": [
+                        "public",
+                        "private",
+                        "research-education",
+                        "limited"
+                    ]
+                },
+                "access_notes": {
+                    "type": "string"
+                },
+                "pscheduler_service": {
+                    "allOf": [
+                        {
+                            "$ref": "#/$definitions/pscheduler_service_schema"
+                        }
+                    ]
+                },
+                "archive_service": {
+                    "$ref": "#/$definitions/archive_service_schema"
+                },
+                "meta": {
+                    "type": "object"
+                }
+            },
+            "unevaluatedProperties": false,
+            "required": [
+                "administrators",
+                "client_uuid",
+                "access_policy",
+                "pscheduler_service",
+                "name"
+            ]
+        },
+        "service": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "service name",
+                    "type": "string"
+                },
+                "locators": {
+                    "description": "service locator uris",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "uri"
+                    }
+                },
+                "version": {
+                    "description": "service version",
+                    "type": "string"
+                },
+                "meta": {
+                    "type": "object"
+                }
+            },
+            "required": [
+                "name",
+                "locators"
+            ]
+        },
+        "pscheduler_service_schema": {
+            "type": "object",
+            "properties": {
+                "tools": {
+                    "description": "pscheduler tools installed",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "tests": {
+                    "description": "pscheduler tests enabled on the host",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/$definitions/service"
+                }
+            ],
+            "unevaluatedProperties": false,
+            "required": [
+                "tools",
+                "tests"
+            ]
+        },
+        "archive_service_schema": {
+            "allOf": [
```

Comment:
We should had the following fields:
- write_urls - A list of urls where results can be stored. For a typical perfSONAR archive results get written to logstash but read from opensearch. We'll assume the default service URL is the read url for opensearch/elasticsearch. This field can be optional. 
- type - this is a required string. Currently will be either opensearch or elasticsearch. this will be helpful when autogenerating Grafana datasoruces since most people run Opensearch, but people like ESnet and CERN run Elasticsearch.